ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml

```diff
@@ -120,19 +130,29 @@ StackLayout {
         title: qsTr("Edit Community")
 
         content: CommunityEditSettingsPanel {
+            id: communityEditSettingsPanel
+
             name: root.name
             description: root.description
             color: root.color
-            image: root.image
+            logoImageData: root.logoImageData
+            bannerImageData: root.bannerImageData
+            isCommunityHistoryArchiveSupportEnabled: root.isCommunityHistoryArchiveSupportEnabled
+            historyArchiveSupportToggle: root.historyArchiveSupportToggle
 
             Component.onCompleted: {
                 editCommunityPage.dirty =
                         Qt.binding(() => {
-                                       return root.name != name ||
-                                              root.description != description ||
-                                              root.image != image ||
-                                              root.color != color
+                                       return root.name !== name ||
+                                              root.description !== description ||
+                                              logoImagePath.length > 0 ||
+                                              root.color !== color ||
+                                              bannerPath.length > 0 ||
+                                              isValidRect(bannerCropRect) ||
+                                              root.historyArchiveSupportToggle !== historyArchiveSupportToggle
```

Comment:
We have a special thing with the Banner and Logo, they come as data images and we generate FS path and cropping rectangle. Therefore we can't and doesn't make sense to compare them. If we have a path and crop rect set it is good to save it. This is from the initial implementation, now I see it could also be optimized only to check for path.